packages/graphql/src/schema/resolvers/composition/wrap-query-and-mutation.ts

```diff
@@ -104,7 +104,10 @@ export const wrapQueryAndMutation =
             cypherQueryOptions: context.cypherQueryOptions,
             sessionConfig: context.sessionConfig,
             cypherParams: context.cypherParams,
-            transactionMetadata: context.transactionMetadata,
+            transaction: {
+                timeout: context.transaction?.timeout,
+                metadata: {...context.transaction?.metadata, ...context.transactionMetadata},
+            },
```

Comment:
```suggestion
            transaction: {
                ...context.transaction
                // TODO: Remove explicit metadata property assignment when transactionMetadata is removed
                metadata: {...context.transaction?.metadata, ...context.transactionMetadata},
            },
```

I think this will allow us to accept anything, even if the `UserTransactionConfig` will need to be maintained. From the perspective of JS, or asserted TS, we would just allow all additional transaction data that may be added in the future to the driver.

Reply:
You are right. Changed. 🫡